docs/dev/cli/cli-commands.md

```diff
@@ -0,0 +1,86 @@
+## Build the CLI from Source
+Steps to build the **TOSCAna Command Line Interface** from sources:
+1. Clone the project.
+
+    `
+    git clone https://github.com/StuPro-TOSCAna/TOSCAna.git
+    `
+
+2. Enter the cli directory.
+
+    `
+    cd TOSCAna/cli
+    `
+
+3. Build the project with:
+
+    `
+    mvn package
+    `
+
+4. You can find the generated jar file `cli-1.0-SNAPSHOT-jar-with-dependencies.jar` in the target folder.
+
+## Usage
+How to use **TOSCAna CLI**:
+
+1. Create an Alias `toscana` on Unix Systems for easier usage:
+
+	`
+    alias toscana='java -jar cli-1.0-SNAPSHOT-jar-with-dependencies.jar'
+    `
+
+2. You can now use the **TOSCAna CLI** with the below commands.
+
+	`
+    toscana [<available commands>]
+    `
+
+## CLI Basics
+
+The main tasks of the CLI component are:
+- parse and call the commands
+- call the methods to transform a given topology to a custom platform
+- show current status of transformation/ deployment
+- print detailed logs if wanted
+- show supported platforms
+
+## Supported Commands, Subcommands and Options
+
+Each of the following commands except help have these options: `-d` or `--debug`, `-u` or `--usage`, `-v` or `--verbose`.
+Parameters of the commands are represented as `<name>`.
+Commands with a command after it are called Subcommands, before you can call a Command or Subcommand you have to type `toscana` and then the command and subcommand.
+
+Options explanation:
+- `debug`: enable Debugging
```

Comment:
The concept of `debug` and `verbose` is super confusing.
In some unix tools I've seen this mechanism:
- -v: verbose
- -vv: more verbose
- -vvv: highest verbosity